src/app/contexts/ToggleContext/reducer.jsx

```diff
@@ -0,0 +1,17 @@
+// toggle actions
+export const updateToggles = data => ({
+  type: 'UPDATE_TOGGLE_STATE',
+  data,
+});
+
+export const toggleReducer = (toggleState, action) => {
+  const { type, data } = action;
+  switch (type) {
+    case 'UPDATE_TOGGLE_STATE':
```

Comment:
will we ever have any other action types? If not then we should consider removing the switch